src/output-json-tls.c

```diff
@@ -190,10 +196,10 @@ static void JsonTlsLogSerial(JsonBuilder *js, SSLState *ssl_state)
     }
 }
 
-static void JsonTlsLogVersion(JsonBuilder *js, SSLState *ssl_state)
+static void JsonTlsLogVersion(JsonBuilder *js, const uint16_t version)
 {
     char ssl_version[SSL_VERSION_MAX_STRLEN];
-    SSLVersionToString(ssl_state->server_connp.version, ssl_version);
```

Comment:
Why this change ?

Reply:
This change was made to apply a bit of DRY to the functionality. Basically, the JA4(S) objects require the version and therefore they're output via EVE as well. Rather than write an identical function to'output' a different version parameter I refactored this function to accept the version, it already accepts the object it's writing to. I hope that's ok?

Reply:
I will take a look again.
My question is if it changes behavior of what gets logged (in edge cases)...